src/pages/Account/AccountPayment.tsx

```diff
@@ -1,10 +1,105 @@
 import * as React from 'react';
+import { ListGroup, ListGroupItem, Row, Col, Modal, ModalHeader, ModalBody, Alert, Button } from 'reactstrap';
+import { Query } from 'react-apollo';
+import { GET_PAYMENT_SOURCES, PaymentSource } from 'networking/paymentSources';
+import { CreditBalance } from 'networking/users';
 
-class AccountPayment extends React.Component<any> {
-  render() {
-    return (
-      <h1>This is the Account Payment Page</h1>
-    );
-  }
+import AudioLoading from 'shared/loading/AudioLoading';
+
+import NewCardForm from './NewCardForm';
+import DeleteCardForm from './DeleteCardForm';
+import { AlertProperties } from 'components/work/Alert/Alert';
+
+interface Props extends RouterProps {
+  creditBalance: CreditBalance;
 }
+
+enum ModalType {
+  ADD_NEW_CARD = 'ADD_NEW_CARD',
+  DELETE_CARD = 'DELETE_CARD',
+}
+
+const AccountPayment = ({ creditBalance }: Props) => {
+  const [modal, setModal] = React.useState<ModalType | undefined>(undefined);
+  const [alert, setAlert] = React.useState<AlertProperties>({ msg: '', color: '', show: false });
+  const [paymentSource, setPaymentSource] = React.useState<PaymentSource | undefined>(undefined);
+  
+  return (
+    <Query query={GET_PAYMENT_SOURCES}>
+      {({ loading, error, data }) => {
+        if (loading) {
+          return <AudioLoading height={48} width={96} />;
+        }
+        if (error || !data) {
+          return <h4>{error ? error.message : 'Error / No Data'}</h4>;
+        }
+        const paymentSources = data.getPaymentSources;
+        const renderPaymentSources = paymentSources.map((paymentSource: PaymentSource) => (
+          <ListGroupItem
+            className="w-100 d-flex justify-content-between align-items-center"
+            id={paymentSource.id}
+            key={paymentSource.id}>
+            <h6 className="mb-0">
+              {paymentSource.stripeBrand}&nbsp;(...{paymentSource.stripeLast4})
+            </h6>
+            <Button
+              className="btn-icon"
```

Comment:
can we make this btn-light so it's an action we don't want to emphasize to the user?

Reply:
👍